test/SecTester.Core.Tests/Logger/DefaultLoggerTests.cs

```diff
@@ -0,0 +1,230 @@
+
+public class DefaultLoggerTests : IDisposable
+{
+  private const string Message = "message";
+  private const string DatetimeFormat = "yyyy-MM-dd HH:mm:ss.fff";
+
+  private static readonly DateTime Moment;
+  private static readonly string HeaderLogLevelError;
+  private static readonly string HeaderLogLevelNotice;
+  private static readonly string HeaderLogLevelWarn;
+  private static readonly string HeaderLogLevelVerbose;
+
+  private readonly ILogAppender _logAppenderMock;
+  private ServiceProvider _serviceProvider;
+
+  static DefaultLoggerTests()
+  {
+    Moment = DateTime.Now;
+    var moment = Moment.ToString(DatetimeFormat, CultureInfo.InvariantCulture);
+
+    HeaderLogLevelError = $"[{moment}] [ERROR  ] - ";
+    HeaderLogLevelNotice = $"[{moment}] [NOTICE ] - ";
+    HeaderLogLevelVerbose = $"[{moment}] [VERBOSE] - ";
+    HeaderLogLevelWarn = $"[{moment}] [WARN   ] - ";
+  }
+
+  public DefaultLoggerTests()
+  {
+    var systemTimeProviderMock = Substitute.For<ISystemTimeProvider>();
+    systemTimeProviderMock.Now.Returns(Moment);
+    _logAppenderMock = Substitute.For<ILogAppender>();
+    _serviceProvider = new ServiceCollection()
+      .AddSingleton(_logAppenderMock)
+      .AddSingleton(systemTimeProviderMock)
+      .AddTransient<DefaultLogger>()
+      .BuildServiceProvider();
+  }
+
+  public void Dispose()
+  {
+    _serviceProvider.Dispose();
+    GC.SuppressFinalize(this);
+  }
+
+  [Fact]
+  public void Constructor_SetALevelToNoticeByDefault()
+  {
+    // act
+    var sut = _serviceProvider.ResolveWith<DefaultLogger>();
+
+    // assert
+    sut.LogLevel.Should().Be(LogLevel.Notice);
+  }
+
+  [Fact]
+  public void Constructor_SetACustomLevel()
+  {
+    // act
+    var sut = _serviceProvider.ResolveWith<DefaultLogger>(LogLevel.Error);
+
+    // assert
+    sut.LogLevel.Should().Be(LogLevel.Error);
+  }
+
+  [Fact]
+  public void LogLevel_ChangeADefaultLevel()
+  {
+    // arrange
+    var sut = _serviceProvider.ResolveWith<DefaultLogger>();
+
+    // act
+    sut.LogLevel = LogLevel.Warn;
+
+    // assert
+    sut.LogLevel.Should().Be(LogLevel.Warn);
+  }
+
+  [Theory]
+  [InlineData(LogLevel.Silent)]
+  [InlineData(LogLevel.Error)]
+  [InlineData(LogLevel.Warn)]
+  [InlineData(LogLevel.Notice)]
+  public void Debug_SkipMessage(LogLevel logLevel)
+  {
+    // arrange
+    var sut = _serviceProvider.ResolveWith<DefaultLogger>(logLevel);
+
+    // act
+    sut.Debug(DefaultLoggerTests.Message);
+
+    // assert
+    _logAppenderMock.DidNotReceiveWithAnyArgs().WriteLine(default, Arg.Any<string>());
+  }
+
+  [Theory]
+  [InlineData(LogLevel.Verbose)]
+  public void Debug_LogMessage(LogLevel logLevel)
+  {
+    // arrange
+    var sut = _serviceProvider.ResolveWith<DefaultLogger>(logLevel);
+
+    // act
+    sut.Debug(DefaultLoggerTests.Message);
+
+    // assert
+    _logAppenderMock.Received(1)
+      .WriteLine(LogLevel.Verbose, $"{DefaultLoggerTests.HeaderLogLevelVerbose}{DefaultLoggerTests.Message}");

```

Comment:
`Message` is an instance member
```suggestion
      .WriteLine(LogLevel.Verbose, $"{DefaultLoggerTests.HeaderLogLevelVerbose}{Message}");

```